### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by ryelo**

### CODE REVIEW

File: src/docs/components/dropdown/README.md [hunk 1]
```diff
@@ -10,10 +10,7 @@ Import Dropdown, its item and styles from this package. Then use them wherever y
 
 ```JSX
 import React from 'react';
-import { Dropdown, DropdownItem } from '@red-hat-insights/insights-frontent-components/components/Dropdown';
-
-import '@red-hat-insights/insights-frontent-components/components/Dropdown.css';
-// in scss file @import '~@red-hat-insights/insights-frontent-components/components/Dropdown.css';
+import { Dropdown, DropdownItem } from '@redhat-cloud-services/frontend-components';
```
Comment (karelhala) [inline src/docs/components/dropdown/README.md]: We should update this file to reflect PF dropdown https://github.com/RedHatInsights/insights-frontend-storybook/issues/79

File: src/docs/api/chrome/DOC.md [hunk 2]
```diff
@@ -35,6 +35,58 @@ App.propTypes = {
...
+            if (!matchPath(location.href, { path: `${document.baseURI}insights/${event.navId}` })) {
+                this.props.history.push(`/${event.navId}`);
+            }
+        });
+    }
+
+    componentDidUpdate(prevProps) {
+        if (this.props !== prevProps) {
+            if (this.props.location.pathname === '/overview') {
+                insights.chrome.appNavClick({ id: 'overview' });
```
Comment (karelhala) [inline src/docs/api/chrome/DOC.md:63]: Thanks for this addition! This will help developers a lot.

**Review by karelhala — DISMISSED** [review]
Looking good!

File: src/docs/components/dropdown/README.md (hunk 1 above)
Comment (ryelo) [reply src/docs/components/dropdown/README.md]: I'm not going to update it entirely, but I will provide a deprecated warning and a link to the new component.

Comment (ryelo) [reply src/docs/components/dropdown/README.md]: Actually, we've removed this component entirely already, so I'll remove this file.

**Review by karelhala — APPROVED** [review]
Looking good!

**Pull request merged**